# LAZER_DISTANCE message data

Header header

float32 min_distance
float32 angle
float32 laser_x
float32 laser_y
